### ISSUE TITLE
AppRate not override preferences

### ISSUE THREAD

**Issue by edumoritz** [pushandplay/cordova-plugin-apprate#290]
I set the preferences of the approute but returns the original modal

**Import:**
`
import { AppRate } from '@ionic-native/app-rate/ngx';
`
**Set Preferences:** 
`this.appRate.preferences = {
      ...this.appRate.preferences,
      useLanguage: 'pt',
      simpleMode: true,
      displayAppName: 'Meu App',
      inAppReview: true,
      usesUntilPrompt: 4,
      promptAgainForEachNewVersion: true,
      storeAppURL: {
        android: "market://details?id=<appstore>"
      },
      customLocale: {
        title: 'Você curtiu o %@?',
        message: 'Se você curtiu o %@ gostaria de nos avaliar?',
        cancelButtonLabel: 'Não, obrigado',
        laterButtonLabel: 'Mais tarde',
        rateButtonLabel: 'Avaliar agora',
        yesButtonLabel: 'Sim',
        noButtonLabel: 'Não',
        appRatePromptMessage: 'teste appr',
        appRatePromptTitle: 'Gostaria de nos avaliar?',
        feedbackPromptMessage: 'teste feed',
        feedbackPromptTitle: 'title feed'
      },
      callbacks: {
        onRateDialogShow: res => console.log('dialog show', res),
        onButtonClicked: button => {
          console.log('Button clicked: ', JSON.stringify(button))
        },
        handleNegativeFeedback: () => console.log('feedback')
      },
      
    }, 
    
    this.appRate.promptForRating(true);
`
**Return:**
![Captura de Tela 2020-10-27 às 16 51 17](https://user-images.githubusercontent.com/19297506/97354954-f6c04200-1874-11eb-9e21-41d68aab5d58.png)
![Captura de Tela 2020-10-27 às 16 51 28](https://user-images.githubusercontent.com/19297506/97354960-f88a0580-1874-11eb-9427-f18128d430ca.png)

**Comment by DorianLovera** [comment]
I had the same issue... 
Before doing : `this.appRate.promptForRating(true);`, you can initialize your preferences :  `this.appRate.preferences = this.preferences;`

Full code 👇
```
 initAppRate() {
    this.platform.ready().then(() => {
      this.appRate.preferences = this.preferences;
      this.appRate.promptForRating(true);
    });
  }
```

**Comment by DorianLovera** [comment]
I ended up downgrading the plugin to v1.6.1 in package.json. 

With the latest version it looks like there is a new method for this purpose : `setPreference()` which isn't supported by the ionic wrapper....

**Comment by NLueg** [comment]
This is fixed with 1.7.1
